metricflow_example_dbt/target/run_results.json

```diff
@@ -0,0 +1 @@
, "timing": [{"name": "compile", "started_at": "2022-08-17T20:17:20.819682Z", "completed_at": "2022-08-17T20:17:20.823297Z"}, {"name": "execute", "started_at": "2022-08-17T20:17:20.823701Z", "completed_at": "2022-08-17T20:17:20.917803Z"}], "thread_id": "Thread-1 (worker)", "execution_time": 0.09927105903625488, "adapter_response": {"_message": "SELECT 4", "code": "SELECT", "rows_affected": 4}, "message": "SELECT 4", "failures": null, "unique_id": "model.transform_dbt_sync.demo_customers_base"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2022-08-17T20:17:20.920615Z", "completed_at": "2022-08-17T20:17:20.923772Z"}, {"name": "execute", "started_at": "2022-08-17T20:17:20.924141Z", "completed_at": "2022-08-17T20:17:20.977305Z"}], "thread_id": "Thread-1 (worker)", "execution_time": 0.057859182357788086, "adapter_response": {"_message": "SELECT 50", "code": "SELECT", "rows_affected": 50}, "message": "SELECT 50", "failures": null, "unique_id": "model.transform_dbt_sync.demo_transactions_base"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2022-08-17T20:17:20.981737Z", "completed_at": "2022-08-17T20:17:20.986932Z"}, {"name": "execute", "started_at": "2022-08-17T20:17:20.987402Z", "completed_at": "2022-08-17T20:17:21.041755Z"}], "thread_id": "Thread-1 (worker)", "execution_time": 0.061353206634521484, "adapter_response": {"_message": "SELECT 26", "code": "SELECT", "rows_affected": 26}, "message": "SELECT 26", "failures": null, "unique_id": "model.transform_dbt_sync.daily_transactions"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2022-08-17T20:17:21.044601Z", "completed_at": "2022-08-17T20:17:21.049721Z"}, {"name": "execute", "started_at": "2022-08-17T20:17:21.050202Z", "completed_at": "2022-08-17T20:17:21.103127Z"}], "thread_id": "Thread-1 (worker)", "execution_time": 0.059664011001586914, "adapter_response": {"_message": "SELECT 4", "code": "SELECT", "rows_affected": 4}, "message": "SELECT 4", "failures": null, "unique_id": "model.transform_dbt_sync.transactions_by_customer"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2022-08-17T20:17:21.106545Z", "completed_at": "2022-08-17T20:17:21.112015Z"}, {"name": "execute", "started_at": "2022-08-17T20:17:21.112562Z", "completed_at": "2022-08-17T20:17:21.163099Z"}], "thread_id": "Thread-1 (worker)", "execution_time": 0.05769014358520508, "adapter_response": {"_message": "SELECT 4", "code": "SELECT", "rows_affected": 4}, "message": "SELECT 4", "failures": null, "unique_id": "model.transform_dbt_sync.transactions_by_customer_with_region"}], "elapsed_time": 0.7126479148864746, "args": {"write_json": true, "use_colors": true, "printer_width": 80, "version_check": true, "partial_parse": true, "static_parser": true, "profiles_dir": "/Users/heshamnawaz/.dbt", "send_anonymous_usage_stats": true, "event_buffer_size": 100000, "quiet": false, "no_print": false, "which": "run", "rpc_method": "run", "indirect_selection": "eager"}}
```

Comment:
I think you can just delete the whole target directory. This will be auto generated when people run this locally.